src/cli/src/error.rs

```diff
@@ -1,84 +1,39 @@
 use std::io::Error as IoError;
 
 use fluvio::FluvioError;
-use fluvio_cluster::{ClusterError, CheckError};
-use fluvio_cluster::extension::ClusterCmdError;
+use fluvio_extension_common::output::OutputError;
 use fluvio_extension_consumer::ConsumerError;
-
-use crate::common::output::OutputError;
+use fluvio_cluster::cli::ClusterCliError;
 use crate::common::target::TargetError;
 
 pub type Result<T> = std::result::Result<T, CliError>;
 
 #[derive(thiserror::Error, Debug)]
 pub enum CliError {
     #[error(transparent)]
-    IoError {
-        #[from]
-        source: IoError,
-    },
+    IoError(#[from] IoError),
     #[error(transparent)]
-    OutputError {
-        #[from]
-        source: OutputError,
-    },
+    OutputError(#[from] OutputError),
+    #[error("Fluvio cluster error")]
+    ClusterCliError(#[from] ClusterCliError),
```

Comment:
Removing these lines causes the project to no longer compile. This is because even though we want to split out fluivo-cluster and fluvio-extension-consumer into independent executables later, they are still directly linked as of now, and therefore we need the `From` conversions from these respective error types. I agree that this needs to eventually happen, but I didn't actually add these error variants in this PR so I believe it's unrelated and should be handled in a future issue.